beta-src/src/components/map/components/WDBuildUnitButtons.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-bitwise */
```

Comment:
Curious what's this for?

Reply:
ESLint automatically prohibits use of bitwise operators to prevent their accidental use since they are uncommon. The return below is using bitwise:
```
return (
    <>
      {buildOptions[canBuild & BuildUnit.Army]}
      {buildOptions[canBuild & BuildUnit.Fleet]}
    </>
  );
  ```